### PR TITLE
Add Galician translations

### PR DESCRIPTION
**Pull request by frandieguez**
Please add Galician translations

### CODE REVIEW

File: po/gl.po [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+"preferencias de privacidade."
+
+#: src/device.c:207
+msgid "An application wants to use your microphone."
+msgstr "Un aplicativo quere usar o seu micrófono."
+
+#: src/device.c:209
+#, c-format
+msgid "%s wants to use your microphone."
+msgstr "%s quere uar o seu micrófono."
```
Comment (mariospr) [inline po/gl.po]: "uar" - > "usar"

File: po/gl.po [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+"preferencias de privacidade."
+
+#: src/device.c:233
+msgid "An application wants to use your camera."
+msgstr "Un aplicativo quere usar a súa cámara."
+
+#: src/device.c:235
+#, c-format
+msgid "%s wants to use your camera."
+msgstr "%s quere uar a súa cámara."
```
Comment (mariospr) [inline po/gl.po]: "uar" -> "usar"

**Comment by frandieguez** [comment]
Typo fixed, thanks!

**Comment by frandieguez** [comment]
Wow, so many errors, what a shame! Now all should be fixed and, as you said, I've squashed all the commits into one.

Hope now everything is ok.

**Comment by mariospr** [comment]
> Wow, so many errors, what a shame! Now all should be fixed and, as you said, I've squashed all the commits into one.

No problem! I guess that's the disadvantage of having a Galician reviewing this translation, since I can spot actual mistakes on it (believe me, my reviews were more relaxed when they were about other languages)

> Hope now everything is ok.

Looks good, and thanks for squashing. Btw, if you could squash those two commits in https://github.com/flatpak/xdg-desktop-portal-gtk/pull/97/commits that would be great too.

**Pull request merged**